src/main/resources/public/js/users.js

```diff
@@ -511,7 +535,7 @@ app.controller('UserDetailsCtrl', function ($scope, $route, $uibModal, $translat
     $scope.userHours = getHours();
 
     // LAST CHANGES
-    $scope.userHistory = API.setUrl("/api/userHistory/recent").get({userMail: $scope.user.principalName});
+    $scope.userHistory = API.setUrl('/api/users/' + $scope.user.id + '/days/recent').query();
```

Comment:
if you change quote marks, change in the whole project.

Reply:
sorry - I fix only what I touch now, not the whole project